.github/workflows/e3sm-gh-ci-cime-tests.yml

```diff
@@ -0,0 +1,64 @@
+name: test
+
+on:
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+
+  run:
+    strategy:
+      fail-fast: false
+      matrix:
+        res: ["ne4pg2_oQU480"]
+        com: ["F2010"]
+        typ: 
+          - "SMS"   # 0h20m
+          - "SMS_D" # 0h30m
+          - "REP"   # 0h30m
+          - "ERS"   # 1h00m
+          - "ERP"   # 1h15m
+          - "ERR"   # 1h00m
+          - "PET"   # 0h35m
+          - "PEM"   # 0h45m
+          - "SEQ"   # 3h10m
+          - "ERI"   # Exception from case_test: [Errno 2] No such file or directory:
+          - "ERT"   # Exception from case_test: ERROR: No variable AVG_HIST_N found in case
+          - "PEA"   # ERROR: BUILD FAIL: build e3sm failed
```

Comment:
please remove theses tests:  ERT (a longer ERS),  ERR (ERS with short-term acrchiving), ERI (hybrid run).  Also PEA (compares 1 task MPI with mpi-serial).  Also remove the SEQ test (have no idea why that's 3 hours but also needs a staggered layout for a fully coupled case to work).